commands/src/main/java/org/wildfly/extras/creaper/commands/infinispan/cache/RemoveCache.java

```diff
@@ -4,11 +4,17 @@
 import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
 import org.wildfly.extras.creaper.core.online.operations.Address;
 import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
 
 public final class RemoveCache implements OnlineCommand {
     private final Address address;
+    private final boolean allowResourceServiceRestart;
 
     public RemoveCache(String cacheContainer, CacheType cacheType, String cacheName) {
+        this(cacheContainer, cacheType, cacheName, false);
+    }
+
+    public RemoveCache(String cacheContainer, CacheType cacheType, String cacheName, boolean allowResourceServiceRestart) {
```

Comment:
Just please add a comment to the class javadoc that with EAP 7.1+, this command enables the service restart header. That would be good enough from my POV. @rhatlapa, WDYT?